player/stream-test/index.html

```diff
@@ -59,6 +59,32 @@
             <input id="drm-license" class="form-control" name="drm-license" type="text" placeholder="DRM License URL">
             <div id="drm-error" class="error"></div>
         </div>
+        <div class="demo-input-box input">
+            <div class="demo-item-header">
+                <div>Test your stream on real devices with Stream Lab</div>
+            </div>
+            <div class="demo-select-target">
+                <select id="available-targets" class="form-control" name="available-targets">
+                    <option value='1'>Chrome on macOS</option>
+                    <option value='2'>Chrome on Ubuntu Linux</option>
+                    <option value='3'>Chrome on Windows 10</option>
+                    <option value='4'>Edge on Windows 10</option>
+                    <option value='5'>Firefox on macOS</option>
+                    <option value='6'>Firefox on Ubuntu Linux</option>
+                    <option value='7'>Firefox on Windows 10</option>
+                    <option value='8'>Internet Explorer 11 on Windows 10</option>
+                    <option value='9'>Internet Explorer 11 on Windows 7</option>
+                    <option value='10'>LG WebOS (various model years)</option>
+                    <option value='11'>Microsoft Xbox Series S</option>
+                    <option value='12'>PlayStation 5</option>
+                    <option value='13'>Safari on macOS</option>
```

Comment:
I think it makes sense to group all entries a bit better. E.g. move Safari to the other browsers, put webos and tizen TVs together.

Reply:
The order comes from the requirements https://bitmovin.slack.com/archives/C017X7MSN2W/p1676897330503529?thread_ts=1676896713.923159&cid=C017X7MSN2W. The same order is on Stream Lab.